subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -568,15 +564,17 @@ static int __wifi_args_to_params(const struct shell *sh, size_t argc, char *argv
 				&params->bssid[4], &params->bssid[5]);
 			break;
 		case 'h':
-			shell_help(sh);
+			return -ENOEXEC;
 			break;
 		default:
 			PR_ERROR("Invalid option %c\n", opt);
 			shell_help(sh);
 			return -EINVAL;
 		}
 	}
-
+	if (params->psk && !secure_connection) {
+		PR_WARNING("Passphrase provided without security configuration\n");
```

Comment:
Should we return error (`-EINVAL`) here or may be specify that since `security` is not configured, setting will default to `NONE` and passphrase will not be used?

Reply:
No when WIFI 6 is added it will use the saepasspharse instead of the passpharase so it would be valid not to have passphrase configured. Also although I cannot think of a reason the user may want a null passphrase. A warning should be sufficient.